### PR TITLE
Feat/es language

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
Add spanish language to easel.

### CODE REVIEW

**Comment by kjawadDeveloper2** [comment]
This PR should be merge after the publish screen PR.

**Comment by heyitsjk** [comment]
For the most part, looks good. Just some translation edits then it’s ready: 

4   "published": "Publicado",

5   "publish_total": "Publicado ({})",

27  "view_on_pylons": "Ver en Pylons",

53  "save": "Guardar",

55   "read_more": "Leer mas",

57   "download_pylons_app": "Descarga la aplicación Pylons",

**Comment by heyitsjk** [comment]
looks good to go!

**Comment by kjawadDeveloper2** [comment]
@mustafapylons @Amean  kindly check only last 4 commits . Since its made from the published nft branch thats why showing old code.

File: lib/screens/choose_format_screen.dart [hunk 1]
```diff
@@ -258,7 +263,8 @@ class _ErrorMessageWidget extends StatelessWidget {
             Column(
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
-                Text("• ${kFileSizeLimitInGB}GB Limit", style: Theme.of(context).textTheme.bodyText2!.copyWith(color: Colors.white, fontSize: 16.sp, fontWeight: FontWeight.w800)),
+                Text((nftTypes == NFTTypes.video || nftTypes == NFTTypes.audio) ? "• ${(kFileSizeLimitForAudiVideoInGB * 1000).toStringAsFixed(0)}MB Limit" : "• ${kFileSizeLimitInGB}GB Limit",
```
Comment (mustafapylons) [inline lib/screens/choose_format_screen.dart]: "1000" can be in a constant.

File: lib/screens/choose_format_screen.dart [hunk 2]
```diff
@@ -313,7 +319,8 @@ class _ErrorMessageWidget extends StatelessWidget {
...
```
Comment (mustafapylons) [inline lib/screens/choose_format_screen.dart]: "1000" can be in a constant.

File: lib/utils/extension_util.dart [hunk 3]
```diff
@@ -112,3 +115,48 @@ extension MyStringSnackBar on String {
...
+
+extension FileExtension on String {
+  String getFileExtension() {
+    return path.extension(this).replaceAll(".", "");
+  }
+}
+
+extension FileSizeInGB on int {
+  double getFileSizeInGB() {
+    return this / (1024 * 1024 * 1024).toDouble();
```
Comment (mustafapylons) [inline lib/utils/extension_util.dart]: "1024" can be in a constant.

**Review by mustafapylons — COMMENTED** [review]
some small fixes, otherwise code is good.

**Pull request closed without merging**